ts/features/settings/devMode/components/DeveloperModeSection.tsx

```diff
@@ -511,7 +513,25 @@ const DeveloperTestEnvironmentSection = ({
   };
 
   const onActiveSessionLoginToggle = (enabled: boolean) => {
-    dispatch(setActiveSessionLoginLocalFlag(enabled));
+    if (enabled) {
+      Alert.alert(
+        "Attenzione",
+        "Per testare questa funzionalità dovrai chiudere e riaprire l'applicazione",
```

Comment:
What about adding strings to the translations file as we did in all other part of this developer section? What do you think?